Testing/functional/tests/cypress/integration/ui/user/dependents.js

```diff
@@ -167,12 +196,26 @@ describe("COVID-19 - Vaccine Proof download", () => {
         cy.intercept("GET", "**/UserProfile/*/Dependent", {
             fixture: "UserProfileService/dependent.json",
         });
-        cy.enableModules([
-            "CovidLabResults",
-            "Laboratory",
-            "Dependent",
-            "Immunization",
-        ]);
+
+        cy.configureSettings({
+            dependents: {
+                enabled: true,
+            },
+            datasets: [
+                {
+                    name: "laboratory",
+                    enabled: true,
+                },
```

Comment:
This dataset can be removed.  It's now "covid19TestResult" which is already enabled here.